examples/powercli/datastore-usage-email/README.md

```diff
@@ -15,6 +15,7 @@ git checkout master
 ```
 
 Step 2 - Update `stack.yml` and `vc-datastore-config.json` with your environment information
+Note: leave SMTP_USERNAME and SMTP_PASSWORD blank if you do not want to use authenticated SMTP
```

Comment:
You need an additional line break between Step 2 and the Note. 

We should probably *bold* Note: